internal/worker.go

```diff
@@ -66,10 +65,8 @@ type (
 		// 1 if needed. For example, set the number to 0.1 means you want your activity to be executed
 		// once for every 10 seconds. This can be used to protect down stream services from flooding.
 		// The zero value of this uses the default value.
-		// default: defaultMaxActivityExecutionRate(100k)
-		// Warning: activity's StartToCloseTimeout starts ticking even if a task is blocked due to
-		// rate limiting.
-		ActivitiesPerSecond float64
+		// default: defaultMaxActivityExecutionRate(100k).
```

Comment:
defaultTaskListActivitiesPerSecond

Reply:
Thinking about it, doesn't make sense for a private variable name to be in public docs, will just remove the name and keep the value.